diff -uprN Newtonsoft.Json-9.0.1.old/Src/Newtonsoft.Json/Newtonsoft.Json.csproj Newtonsoft.Json-9.0.1/Src/Newtonsoft.Json/Newtonsoft.Json.csproj
--- Newtonsoft.Json-9.0.1.old/Src/Newtonsoft.Json/Newtonsoft.Json.csproj	2016-06-13 14:03:56.000000000 +0300
+++ Newtonsoft.Json-9.0.1/Src/Newtonsoft.Json/Newtonsoft.Json.csproj	2016-07-03 10:06:01.283051870 +0300
@@ -11,6 +11,8 @@
     <AssemblyName>Newtonsoft.Json</AssemblyName>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
     <FileAlignment>512</FileAlignment>
+	<SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\..\json.net.custom.signkey.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
@@ -18,7 +20,7 @@
     <Optimize>false</Optimize>
     <OutputPath>bin\Debug\Net45\</OutputPath>
     <IntermediateOutputPath>obj\Debug\Net45\</IntermediateOutputPath>
-    <DefineConstants>TRACE;DEBUG;CODE_ANALYSIS</DefineConstants>
+    <DefineConstants>TRACE;DEBUG;CODE_ANALYSIS;SIGNED</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Debug\Net45\Newtonsoft.Json.xml</DocumentationFile>
@@ -31,7 +33,7 @@
     <Optimize>true</Optimize>
     <OutputPath>bin\Release\Net45\</OutputPath>
     <IntermediateOutputPath>obj\Release\Net45\</IntermediateOutputPath>
-    <DefineConstants>TRACE;CODE_ANALYSIS</DefineConstants>
+    <DefineConstants>TRACE;CODE_ANALYSIS;SIGNED</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <DocumentationFile>bin\Release\Net45\Newtonsoft.Json.xml</DocumentationFile>
@@ -262,4 +264,4 @@
   <Target Name="AfterBuild">
   </Target>
   -->
-</Project>
\ No newline at end of file
+</Project>
