diff -uprN nunit.old/build.cake nunit.new/build.cake
--- nunit.old/build.cake	2016-07-10 17:13:48.660384512 +0300
+++ nunit.new/build.cake	2016-07-10 17:36:48.005715770 +0300
@@ -627,60 +627,6 @@ Task("PackageNuGet")
 		PackageRunnerWithExtensions("nuget/runners/nunit.console-runner-with-extensions.nuspec", packageVersion, teamcityVersion, currentImageDir, PACKAGE_DIR);
 
 		PackageRunnerWithExtensions("nuget/runners/nunit.runners.nuspec", packageVersion, teamcityVersion, currentImageDir, PACKAGE_DIR);
-
-        // Package engine
-        NuGetPack("nuget/engine/nunit.engine.nuspec", new NuGetPackSettings()
-        {
-            Version = packageVersion,
-            BasePath = currentImageDir,
-            OutputDirectory = PACKAGE_DIR,
-            NoPackageAnalysis = true
-        });
-        NuGetPack("nuget/engine/nunit.engine.tool.nuspec", new NuGetPackSettings()
-        {
-            Version = packageVersion,
-            BasePath = currentImageDir,
-            OutputDirectory = PACKAGE_DIR,
-            NoPackageAnalysis = true
-        });
-
-        // Package Extensions
-        NuGetPack("nuget/extensions/nunit-project-loader.nuspec", new NuGetPackSettings()
-        {
-            Version = packageVersion,
-            BasePath = currentImageDir,
-            OutputDirectory = PACKAGE_DIR,
-            NoPackageAnalysis = true
-        });
-        NuGetPack("nuget/extensions/vs-project-loader.nuspec", new NuGetPackSettings()
-        {
-            Version = packageVersion,
-            BasePath = currentImageDir,
-            OutputDirectory = PACKAGE_DIR,
-            NoPackageAnalysis = true
-        });
-        NuGetPack("nuget/extensions/nunit-v2-result-writer.nuspec", new NuGetPackSettings()
-        {
-            Version = packageVersion,
-            BasePath = currentImageDir,
-            OutputDirectory = PACKAGE_DIR,
-            NoPackageAnalysis = true
-        });
-        NuGetPack("nuget/extensions/nunit.v2.driver.nuspec", new NuGetPackSettings()
-        {
-            Version = packageVersion,
-            BasePath = currentImageDir,
-            OutputDirectory = PACKAGE_DIR,
-            NoPackageAnalysis = true
-        });
-        NuGetPack("nuget/extensions/teamcity-event-listener.nuspec", new NuGetPackSettings()
-        {
-		    // The teamcity-event-listener extension uses its own versioning
-            Version = teamcityVersion,
-            BasePath = currentImageDir,
-            OutputDirectory = PACKAGE_DIR,
-            NoPackageAnalysis = true
-        });
     });
 
 Task("PackageMsi")
@@ -868,7 +814,6 @@ void PackageRunnerWithExtensions(string
 
 Task("Build")
   .IsDependentOn("BuildAllFrameworks")
-  .IsDependentOn("BuildEngine")
   .IsDependentOn("BuildConsole");
 
 Task("Rebuild")
@@ -878,13 +823,7 @@ Task("Rebuild")
 Task("BuildAllFrameworks")
     .IsDependentOn("InitializeBuild")
     .IsDependentOn("Build45")
-    .IsDependentOn("Build40")
-    .IsDependentOn("Build35")
-    .IsDependentOn("Build20")
-// NOTE: The following tasks use Criteria and will be skipped on Linux
-    .IsDependentOn("BuildPortable")
-    .IsDependentOn("BuildSL")
-    .IsDependentOn("BuildCF");
+    .IsDependentOn("Build40");
 
 Task("TestAll")
     .IsDependentOn("TestAllFrameworks")
diff -uprN nunit.old/nuget/framework/nunit.nuspec nunit.new/nuget/framework/nunit.nuspec
--- nunit.old/nuget/framework/nunit.nuspec	2016-07-10 17:13:48.660384512 +0300
+++ nunit.new/nuget/framework/nunit.nuspec	2016-07-10 17:21:50.161812767 +0300
@@ -32,11 +32,6 @@ Supported platforms:
     <file src="LICENSE.txt" />
     <file src="NOTICES.txt" />
     <file src="CHANGES.txt" />
-    <file src="bin/net-2.0/nunit.framework.dll" target="lib/net20" />
-    <file src="bin/net-2.0/nunit.framework.xml" target="lib/net20" />
-    <file src="bin/net-2.0/NUnit.System.Linq.dll" target="lib/net20" />
-    <file src="bin/net-3.5/nunit.framework.dll" target="lib/net35" />
-    <file src="bin/net-3.5/nunit.framework.xml" target="lib/net35" />
     <file src="bin/net-4.0/nunit.framework.dll" target="lib/net40" />
     <file src="bin/net-4.0/nunit.framework.xml" target="lib/net40" />
     <file src="bin/net-4.5/nunit.framework.dll" target="lib/net45" />
diff -uprN nunit.old/nuget/nunitlite/nunitlite.nuspec nunit.new/nuget/nunitlite/nunitlite.nuspec
--- nunit.old/nuget/nunitlite/nunitlite.nuspec	2016-07-10 17:13:48.660384512 +0300
+++ nunit.new/nuget/nunitlite/nunitlite.nuspec	2016-07-10 17:22:13.421812717 +0300
@@ -36,12 +36,8 @@ How to use this package:
     <file src="LICENSE.txt" />
     <file src="NOTICES.txt" />
     <file src="CHANGES.txt" />
-    <file src="bin/net-2.0/nunitlite.dll" target="lib/net20" />
-    <file src="bin/net-3.5/nunitlite.dll" target="lib/net35" />
     <file src="bin/net-4.0/nunitlite.dll" target="lib/net40" />
     <file src="bin/net-4.5/nunitlite.dll" target="lib/net45" />
-    <file src="bin/net-2.0/nunitlite-runner.exe" target="tools/net20" />
-    <file src="bin/net-3.5/nunitlite-runner.exe" target="tools/net35" />
     <file src="bin/net-4.0/nunitlite-runner.exe" target="tools/net40" />
     <file src="bin/net-4.5/nunitlite-runner.exe" target="tools/net45" />
     <file src="../../nuget/nunitlite/Program.cs" target="content" />
