diff -uprN nunit/build.cake nunit.new/build.cake
--- nunit/build.cake	2016-07-10 15:48:08.575832597 +0300
+++ nunit.new/build.cake	2016-07-10 16:29:25.806637433 +0300
@@ -602,12 +602,7 @@ Task("PackageNuGet")
             BasePath = currentImageDir,
             OutputDirectory = PACKAGE_DIR
         });
-        NuGetPack("nuget/framework/nunitSL.nuspec", new NuGetPackSettings()
-        {
-            Version = packageVersion,
-            BasePath = currentImageDir,
-            OutputDirectory = PACKAGE_DIR
-        });
+
 
         // Package NUnitLite
         NuGetPack("nuget/nunitlite/nunitlite.nuspec", new NuGetPackSettings()
@@ -616,12 +611,7 @@ Task("PackageNuGet")
             BasePath = currentImageDir,
             OutputDirectory = PACKAGE_DIR
         });
-        NuGetPack("nuget/nunitlite/nunitliteSL.nuspec", new NuGetPackSettings()
-        {
-            Version = packageVersion,
-            BasePath = currentImageDir,
-            OutputDirectory = PACKAGE_DIR
-        });
+
 
         // Package Runners
         NuGetPack("nuget/runners/nunit.console-runner.nuspec", new NuGetPackSettings()
diff -uprN nunit/nuget/engine/nunit.engine.nuspec nunit.new/nuget/engine/nunit.engine.nuspec
--- nunit/nuget/engine/nunit.engine.nuspec	2016-07-10 15:47:31.551832677 +0300
+++ nunit.new/nuget/engine/nunit.engine.nuspec	2016-07-10 16:52:20.324232701 +0300
@@ -21,13 +21,13 @@
     <file src="LICENSE.txt" />
     <file src="NOTICES.txt" />
     <file src="CHANGES.txt" />
-    <file src="bin\nunit.engine.dll" target="lib" />
-    <file src="bin\nunit.engine.api.dll" target="lib" />
-    <file src="bin\Mono.Cecil.dll" target="lib" />
-    <file src="bin\nunit-agent.exe" target="lib" />
-    <file src="bin\nunit-agent.exe.config" target="lib" />
-    <file src="bin\nunit-agent-x86.exe" target="lib" />
-    <file src="bin\nunit-agent-x86.exe.config" target="lib" />
-    <file src="..\..\nuget\engine\nunit.nuget.addins" target="lib"/>
+    <file src="bin/nunit.engine.dll" target="lib" />
+    <file src="bin/nunit.engine.api.dll" target="lib" />
+    <file src="bin/Mono.Cecil.dll" target="lib" />
+    <file src="bin/nunit-agent.exe" target="lib" />
+    <file src="bin/nunit-agent.exe.config" target="lib" />
+    <file src="bin/nunit-agent-x86.exe" target="lib" />
+    <file src="bin/nunit-agent-x86.exe.config" target="lib" />
+    <file src="../../nuget/engine/nunit.nuget.addins" target="lib"/>
   </files>
-</package>
\ No newline at end of file
+</package>
diff -uprN nunit/nuget/engine/nunit.engine.tool.nuspec nunit.new/nuget/engine/nunit.engine.tool.nuspec
--- nunit/nuget/engine/nunit.engine.tool.nuspec	2016-07-10 15:47:31.551832677 +0300
+++ nunit.new/nuget/engine/nunit.engine.tool.nuspec	2016-07-10 16:53:38.988232532 +0300
@@ -21,14 +21,14 @@
     <file src="LICENSE.txt" />
     <file src="NOTICES.txt" />
     <file src="CHANGES.txt" />
-    <file src="bin\nunit.engine.api.dll" target="lib" />
-    <file src="bin\nunit.engine.api.dll" target="tools" />
-    <file src="bin\nunit.engine.dll" target="tools" />
-    <file src="bin\Mono.Cecil.dll" target="tools" />
-    <file src="bin\nunit-agent.exe" target="tools" />
-    <file src="bin\nunit-agent.exe.config" target="tools" />
-    <file src="bin\nunit-agent-x86.exe" target="tools" />
-    <file src="bin\nunit-agent-x86.exe.config" target="tools" />
-    <file src="..\..\nuget\engine\nunit.nuget.addins" target="tools"/>  
+    <file src="bin/nunit.engine.api.dll" target="lib" />
+    <file src="bin/nunit.engine.api.dll" target="tools" />
+    <file src="bin/nunit.engine.dll" target="tools" />
+    <file src="bin/Mono.Cecil.dll" target="tools" />
+    <file src="bin/nunit-agent.exe" target="tools" />
+    <file src="bin/nunit-agent.exe.config" target="tools" />
+    <file src="bin/nunit-agent-x86.exe" target="tools" />
+    <file src="bin/nunit-agent-x86.exe.config" target="tools" />
+    <file src="../../nuget/engine/nunit.nuget.addins" target="tools"/>  
   </files>
-</package>
\ No newline at end of file
+</package>
diff -uprN nunit/nuget/extensions/nunit-project-loader.nuspec nunit.new/nuget/extensions/nunit-project-loader.nuspec
--- nunit/nuget/extensions/nunit-project-loader.nuspec	2016-07-10 15:47:31.551832677 +0300
+++ nunit.new/nuget/extensions/nunit-project-loader.nuspec	2016-07-10 16:55:50.650528894 +0300
@@ -20,6 +20,6 @@
   <files>
     <file src="LICENSE.txt" />
     <file src="NOTICES.txt" />
-    <file src="bin\addins\nunit-project-loader.dll" target="tools"/>
+    <file src="bin/addins/nunit-project-loader.dll" target="tools"/>
   </files>
 </package>
diff -uprN nunit/nuget/extensions/nunit.v2.driver.nuspec nunit.new/nuget/extensions/nunit.v2.driver.nuspec
--- nunit/nuget/extensions/nunit.v2.driver.nuspec	2016-07-10 15:47:31.551832677 +0300
+++ nunit.new/nuget/extensions/nunit.v2.driver.nuspec	2016-07-10 16:55:38.190528921 +0300
@@ -20,9 +20,9 @@
   <files>
     <file src="LICENSE.txt" />
     <file src="NOTICES.txt" />
-    <file src="bin\addins\nunit.v2.driver.dll" target="tools"/>
-    <file src="bin\addins\nunit.core.dll" target="tools"/>
-    <file src="bin\addins\nunit.core.interfaces.dll" target="tools"/>
-    <file src="..\..\nuget\extensions\nunit.v2.driver.addins" target="tools"/>
+    <file src="bin/addins/nunit.v2.driver.dll" target="tools"/>
+    <file src="bin/addins/nunit.core.dll" target="tools"/>
+    <file src="bin/addins/nunit.core.interfaces.dll" target="tools"/>
+    <file src="../../nuget/extensions/nunit.v2.driver.addins" target="tools"/>
   </files>
 </package>
diff -uprN nunit/nuget/extensions/nunit-v2-result-writer.nuspec nunit.new/nuget/extensions/nunit-v2-result-writer.nuspec
--- nunit/nuget/extensions/nunit-v2-result-writer.nuspec	2016-07-10 15:47:31.551832677 +0300
+++ nunit.new/nuget/extensions/nunit-v2-result-writer.nuspec	2016-07-10 16:56:00.686528872 +0300
@@ -20,6 +20,6 @@
   <files>
     <file src="LICENSE.txt" />
     <file src="NOTICES.txt" />
-    <file src="bin\addins\nunit-v2-result-writer.dll" target="tools"/>
+    <file src="bin/addins/nunit-v2-result-writer.dll" target="tools"/>
   </files>
 </package>
diff -uprN nunit/nuget/extensions/teamcity-event-listener.nuspec nunit.new/nuget/extensions/teamcity-event-listener.nuspec
--- nunit/nuget/extensions/teamcity-event-listener.nuspec	2016-07-10 15:47:31.551832677 +0300
+++ nunit.new/nuget/extensions/teamcity-event-listener.nuspec	2016-07-10 16:56:18.726528833 +0300
@@ -20,6 +20,6 @@
   <files>
     <file src="LICENSE.txt" />
     <file src="NOTICES.txt" />
-    <file src="bin\addins\teamcity-event-listener.dll" target="tools"/>
+    <file src="bin/addins/teamcity-event-listener.dll" target="tools"/>
   </files>
 </package>
diff -uprN nunit/nuget/extensions/vs-project-loader.nuspec nunit.new/nuget/extensions/vs-project-loader.nuspec
--- nunit/nuget/extensions/vs-project-loader.nuspec	2016-07-10 15:47:31.551832677 +0300
+++ nunit.new/nuget/extensions/vs-project-loader.nuspec	2016-07-10 16:56:37.050528794 +0300
@@ -20,6 +20,6 @@
   <files>
     <file src="LICENSE.txt" />
     <file src="NOTICES.txt" />
-    <file src="bin\addins\vs-project-loader.dll" target="tools"/>
+    <file src="bin/addins/vs-project-loader.dll" target="tools"/>
   </files>
 </package>
diff -uprN nunit/nuget/framework/nunit.nuspec nunit.new/nuget/framework/nunit.nuspec
--- nunit/nuget/framework/nunit.nuspec	2016-07-10 15:47:31.551832677 +0300
+++ nunit.new/nuget/framework/nunit.nuspec	2016-07-10 16:36:43.214549780 +0300
@@ -41,9 +41,5 @@ Supported platforms:
     <file src="bin/net-4.0/nunit.framework.xml" target="lib/net40" />
     <file src="bin/net-4.5/nunit.framework.dll" target="lib/net45" />
     <file src="bin/net-4.5/nunit.framework.xml" target="lib/net45" />
-    <file src="bin/portable/nunit.framework.dll" target="lib/dotnet" />
-    <file src="bin/portable/nunit.framework.xml" target="lib/dotnet" />
-    <file src="bin/portable/nunit.framework.dll" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac+MonoAndroid10+MonoTouch10+Xamarin.iOS10" />
-    <file src="bin/portable/nunit.framework.xml" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac+MonoAndroid10+MonoTouch10+Xamarin.iOS10" />
   </files>
 </package>
diff -uprN nunit/nuget/nunitlite/nunitlite.nuspec nunit.new/nuget/nunitlite/nunitlite.nuspec
--- nunit/nuget/nunitlite/nunitlite.nuspec	2016-07-10 15:47:31.551832677 +0300
+++ nunit.new/nuget/nunitlite/nunitlite.nuspec	2016-07-10 16:41:12.006661848 +0300
@@ -40,15 +40,12 @@ How to use this package:
     <file src="bin/net-3.5/nunitlite.dll" target="lib/net35" />
     <file src="bin/net-4.0/nunitlite.dll" target="lib/net40" />
     <file src="bin/net-4.5/nunitlite.dll" target="lib/net45" />
-    <file src="bin/portable/nunitlite.dll" target="lib/dotnet" />
-    <file src="bin/portable/nunitlite.dll" target="lib/portable-net45+win8+wp8+wpa81+Xamarin.Mac" />
     <file src="bin/net-2.0/nunitlite-runner.exe" target="tools/net20" />
     <file src="bin/net-3.5/nunitlite-runner.exe" target="tools/net35" />
     <file src="bin/net-4.0/nunitlite-runner.exe" target="tools/net40" />
     <file src="bin/net-4.5/nunitlite-runner.exe" target="tools/net45" />
-    <file src="bin/portable/nunitlite-runner.exe" target="tools/portable" />
-    <file src="..\..\nuget\nunitlite\Program.cs" target="content" />
-    <file src="..\..\nuget\nunitlite\Program.vb" target="content" />
-    <file src="..\..\nuget\nunitlite\install.ps1" target="tools" />
+    <file src="../../nuget/nunitlite/Program.cs" target="content" />
+    <file src="../../nuget/nunitlite/Program.vb" target="content" />
+    <file src="../../nuget/nunitlite/install.ps1" target="tools" />
   </files>
 </package>
diff -uprN nunit/nuget/runners/nunit.console-runner.nuspec nunit.new/nuget/runners/nunit.console-runner.nuspec
--- nunit/nuget/runners/nunit.console-runner.nuspec	2016-07-10 15:47:31.551832677 +0300
+++ nunit.new/nuget/runners/nunit.console-runner.nuspec	2016-07-10 16:46:31.729356805 +0300
@@ -35,6 +35,6 @@
     <file src="bin/nunit.engine.api.xml" target="tools" />
     <file src="bin/nunit.engine.dll" target="tools" />
     <file src="bin/Mono.Cecil.dll" target="tools" />
-    <file src="..\..\nuget\engine\nunit.nuget.addins" target="tools"/>
+    <file src="../../nuget/engine/nunit.nuget.addins" target="tools"/>
   </files>
 </package>
